tests/integration/components/user-skills-input-test.js

```diff
@@ -33,53 +37,24 @@ moduleForComponent('user-skills-input', 'Integration | Component | user skills i
   beforeEach() {
     stubService(this, 'store', mockStore);
     stubService(this, 'user-skills', mockUserSkillsService);
+    page.setContext(this);
```

Comment:
I added this in based on what the error on the test logs suggested, but I didn't quite understand what happened here?

The tests pass with this.